src/handlers/ask-llm.ts

```diff
@@ -19,61 +44,44 @@ export async function askQuestion(context: Context, question: string) {
   if (!question) {
     throw context.logger.error("No question provided");
   }
-  const { specAndBodies, streamlinedComments } = await recursivelyFetchLinkedIssues({
-    context,
-    owner: context.payload.repository.owner.login,
-    repo: context.payload.repository.name,
+
+  const response = await askGpt(context, question);
+  context.logger.info(`Answer: ${response.answer}`, {
+    caller: "_Logs.<anonymous>",
```

Comment:
What is this

Reply:
this is added via the logger you don't need `caller`